controller/nc-news-controller.js

```diff
@@ -11,5 +11,12 @@ function sendTopics(req, res, next) {
     }).catch(next)
 }
 
+function sendArticle(req , res, next) {
+    const { article_id } = req.params
+    return fetchArticle(article_id).then((article) =>{
+        res.status(200).send( article )
```

Comment:
Any data sent from a controller needs to be sent back on a meaningful key ie. .send({article})